extension/src/ui/components/VideoDataSyncDialaog.js

```diff
@@ -0,0 +1,119 @@
+import Button from '@material-ui/core/Button';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Grid from '@material-ui/core/Grid';
+import MenuItem from '@material-ui/core/MenuItem';
+import TextField from '@material-ui/core/TextField';
+import { makeStyles } from '@material-ui/styles';
+import React, { useEffect, useState } from 'react';
+
+const createClasses = makeStyles((theme) => ({
+    relative: {
+        position: 'relative',
+    },
+    spinner: {
```

Comment:
Sorry for being so OCD, but on my computer the spinner seems a little bit off center. The spinner should be in line with the arrow icon, and the arrow icon should be the same distance away from the spinner as it is from the edge of the box.
![asbplayer-pr-offcenter-20220115](https://user-images.githubusercontent.com/2254858/149630579-3791f28e-2373-414a-ba17-200d69b304b4.jpg)
: